test/evals/eval_data.yaml

```diff
@@ -1,62 +1,79 @@
       description: Cluster creation request test using keyword matching for reliability
 
 - conversation_group: list_versions_conv
-  description: Test basic tool calling with predictable parameters
+  tags: [smoke]
+  description: "OpenShift versions listing test"
   conversation:
     - eval_id: list_versions_tool_call
       eval_query: What OpenShift versions are available?
-      eval_types: [tool_eval, response_eval:sub-string]
+      eval_types: [tool_eval, response_eval:sub-string, response_eval:intent]
       expected_tool_calls:
         - - tool_name: list_versions
             arguments: {}
       expected_keywords: ["versions", "available", "OpenShift"]
+      expected_intent: "Providing a list of available OpenShift versions."
       description: Test single tool call with no dynamic parameters
 
 - conversation_group: available_operators_conv
+  tags: [smoke]
+  description: "Available operators test"
   conversation:
     - eval_id: available_operators
       eval_query: What operators are available?
-      eval_types: [response_eval:accuracy, tool_eval]
+      eval_types: [response_eval:accuracy, tool_eval, response_eval:intent]
       expected_response: "The operators that can be installed onto clusters are OpenShift AI and OpenShift Virtualization."
       expected_tool_calls:
         - - tool_name: list_operator_bundles
             arguments: {}
+      expected_intent: "Listing the operators that can be installed onto clusters, OpenShift AI and OpenShift Virtualization."
 
 - conversation_group: sno_requirements_conv
+  tags: [smoke]
+  description: "Single Node OpenShift hardware requirements test"
   conversation:
     - eval_id: sno_requirements
       eval_query: What are the host requirements for a single node cluster?
-      eval_types: [response_eval:accuracy]
+      eval_types: [response_eval:accuracy, response_eval:intent]
       expected_response: "A single node cluster requires 8 CPU cores 16 GB of RAM and 100 GB of storage"
+      expected_intent: "Providing hardware requirements for Single Node OpenShift (SNO) clusters including CPU, RAM, and storage specifications."
 
 - conversation_group: multinode_requirements_conv
+  tags: [smoke]
+  description: "Multi-node hardware requirements test"
   conversation:
     - eval_id: multinode_requirements
       eval_query: What are the host requirements for a multinode cluster?
-      eval_types: [response_eval:accuracy]
+      eval_types: [response_eval:accuracy, response_eval:intent]
       expected_response: "Control plane nodes in a multi-node cluster require 4 CPU cores 16 GB of RAM and 100 GB of storage, the compute nodes in this cluster require 2 CPU cores, 8 GB of RAM and 100GB of storage"
+      expected_intent: "Providing hardware requirements for multi-node OpenShift clusters including CPU, RAM, and storage specifications for control plane and worker nodes."
```

Comment:
The question is about `multinode cluster` so the reply is about `multinode cluster` and not 
`multi-node OpenShift clusters` which make the: response_eval:intent: FAIL!

See: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/rh-ecosystem-edge_assisted-chat/246/pull-ci-rh-ecosystem-edge-assisted-chat-main-eval-test/1990357108888965120